news/2022-09-01-how-zetera-became-a-medal-hunter.md

```diff
@@ -0,0 +1,55 @@
+
+In October of 2017 a new [Halloween themed update](https://osu.ppy.sh/home/news/2017-10-31-halloween-descends) was released for osu! and 9 new hush-hush medals were added. At that point it has been a year since a fresh batch of hush-hush medals were added into the game and finally, the community had something to work through and discover the solutions to. 
+
+Since getting batches of medals was still new, there wasn't a lot of knowledge behind what made the medal system tick, especially because the whole system was rewritten in 2016. Nowadays the community knows exactly what to look for and how extensive the system is, but back then the community's ideas were much more abstract than what was actually possible, simply because it was really hard to gauge just how precise requirements were for any given medal. The only hint that was provided was in the flavor text for the medal that is viewable once the cursor is hovered over the medal, but even that can sometimes be considered unhelpful in figuring out the solution and some have been changed multiple times over the years.
+
+Despite all of the above, the community managed to figure out the solutions to all of the new medals! Well, all but one.. "The Girl in the Forest" stood unclaimed. They deduced that the map that needed to be played was [Takuya](https://osu.ppy.sh/users/342372)'s set of [Pika Girl](https://osu.ppy.sh/beatmapsets/40440#osu/128718) by [S3RL](https://osu.ppy.sh/beatmaps/artists/9). Zetera immediately went and SS'd the map, but that didn't work. Maybe a mod was required, or the map needed to be FC'd in all game modes consecutively to get the medal, but that didn't work either. Maybe it had to be an SS? He started thinking about how the system could work, what would it look for? Surely it can't check when you lost your full combo or when an SS was lost, so the only things left would be the miss count, max combo, counts of 300s, 100s, 50s and misses, playing in different modes and a couple of other things, such as at what time was the score submitted. Looking at the flavor text it read out the following: "First Generation in more than one ways", that lead him to think that it might be Pokémon related or perhaps the first generation of influential mappers, in the end it wasn't much of a help to him. Out of ideas, he reached out to [Ephemeral](https://osu.ppy.sh/users/102335), osu!'s "community manager" and instead of receiving a hint, Ephemeral would go on to create the [following thread](https://osu.ppy.sh/community/forums/topics/662279?n=1) to the osu! forum:
+
+> It's been roughly a week since the Halloween medal release, and just about every medal outside of one has been conclusively claimed and figured out.
+> Originally, I thought that the Firmament Moves would be the baffler for most people. Apparently not!
+> So, let's have a little fun with this, and up the ante a little bit.
```

Comment:
i'm pretty sure these paragraphs should be spaced like so but i also remember that remark complains about it <.< (@cl8n ?)
```
> ...
> 
> ...
>
> ...
```

Reply:
Discussed on [Discord ](https://discord.com/channels/188630481301012481/218677502141399041/1016278123056218182)and applied via [816c1f7](https://github.com/ppy/osu-wiki/pull/7949/commits/816c1f78326548bd1506e6a0976cf9f42b46e469)